dev/fattest.simplicity/src/componenttest/topology/impl/LibertyServer.java

```diff
@@ -3190,6 +3212,10 @@ public ProgramOutput stopServer(boolean postStopServerArchive, boolean forceStop
         return output;
     }
 
+    protected void checkLogsForErrorsAndWarnings(String... ignoredFailuresRegExps) throws Exception {
+        checkLogsForErrorsAndWarnings(Arrays.asList(LIBERTY_ERROR_REGEX), ignoredFailuresRegExps);
+    }
+
```

Comment:
Should this method actually be removed or deprecated?

I would have thought any code should now be calling the new method.

Reply:
This is the one calling `stopServer()` without arguments ends up at. I think we want to keep no-args as the default.